homeassistant/components/flume/sensor.py

```diff
@@ -135,72 +138,69 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        return self._state
+        sensor_key = self._flume_query_sensor[0]
+        if sensor_key not in self._flume_device.values:
+            _LOGGER.debug(
```

Comment:
We don't allow side effects like logging in entity state properties.

Reply:
Removed here : https://github.com/home-assistant/core/pull/42126